### PR TITLE
Feature#1/login page admin

### LINKED ISSUE #1: Login page
Design with HTML, Bootstrap and CSS for the login page for the admin login

- Form with two fields
- A field for user of the admin
- A field for the password of the admin
- A login button

### PR DESCRIPTION
**Pull request by javs1**
# Description
What?:
We created the design and front-end for the login page using CSS, JS, HTML and Bootstrap, the page desing includes a bootstrap form with 2 text inputs and a button to submit the inputs

Why?:
We need the design of the login page as a base for the app to create the front-end of the other pages.

### CODE REVIEW

File: Reloj_checador/app/assets/stylesheets/home.scss [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
-    background-image: url('https://mcdn.wallpapersafari.com/medium/37/48/dxjKpR.jpg');
+    background-color: #242236 !important;
     background-size: cover;
     background-repeat: no-repeat;
     height: 100%;
     font-family: 'Numans', sans-serif;
     }
     
-    .container{
+    .container2{
```
Comment (kevinnio) [inline Reloj_checador/app/assets/stylesheets/home.scss]: I'm not a fan of `thing1, thing2, thing3...` naming schemes. How about using a name that tells us more about what the container's purpose instead? Something like `main-container` or `content-container`.

File: Reloj_checador/app/assets/stylesheets/home.scss [hunk 2]
```diff
@@ -68,13 +68,12 @@ html,body{
     
     .login_btn{
     color: black;
-    background-color: #FFC312;
     width: 100px;
+    background-color: white !important;
```
Comment (kevinnio) [inline Reloj_checador/app/assets/stylesheets/home.scss]: Using `!important` may cause issues if you want to override styles later on. Is it really needed here? Maybe this works without it. 🤔

File: Reloj_checador/app/views/home/index.html.erb [hunk 3]
```diff
@@ -6,7 +6,7 @@
     </head>
 
     <body>
-    <div class="container">
+    <div class="container container2">
```
Comment (kevinnio) [inline Reloj_checador/app/views/home/index.html.erb]: Does this element needs these two CSS classes? I think it would be fine with only one of them. 🤔

**Review by kevinnio — CHANGES_REQUESTED** [review]
Looking good, but it can be improved.

File: Reloj_checador/app/assets/stylesheets/home.scss (hunk 1 above)
Comment (BrandonVG) [reply Reloj_checador/app/assets/stylesheets/home.scss]: Changed the name of the container2 to login-container.

File: Reloj_checador/app/assets/stylesheets/home.scss (hunk 2 above)
Comment (BrandonVG) [reply Reloj_checador/app/assets/stylesheets/home.scss]: Deleted the !important

File: Reloj_checador/app/views/home/index.html.erb (hunk 3 above)
Comment (BrandonVG) [reply Reloj_checador/app/views/home/index.html.erb]: Deleted the container class and changed container2 with login-container class.

File: Reloj_checador/app/assets/stylesheets/home.scss [hunk 4]
```diff
@@ -7,7 +7,8 @@ html,body{
     font-family: 'Numans', sans-serif;
     }
     
-    .container2{
+    .login-container{
+    margin-top: 10rem;
```
Comment (xFerchoVJ) [inline Reloj_checador/app/assets/stylesheets/home.scss]: I recommend that you put the properties of your styles alphabetically, it is a good practice.

File: Reloj_checador/app/assets/stylesheets/home.scss [hunk 5]
```diff
@@ -51,34 +37,12 @@ html,body{
...
-    
-    .links{
-    color: white;
-    }
-    
-    .links a{
-    margin-left: 4px;
-    }
\ No newline at end of file
+
```
Comment (xFerchoVJ) [inline Reloj_checador/app/assets/stylesheets/home.scss]: Remember to add a blank line at the end of the file, when you submit the PR github notify you about that with this sign.

**Review by kevinnio — APPROVED** [review]
Thanks for addressing my comments.

**Review by xFerchoVJ — APPROVED** [review]
LGTM

**Pull request merged**